requirements.txt

```diff
@@ -0,0 +1,3 @@
+"spacy>=3.2.0,<3.5.0"
```

Comment:
```suggestion
spacy>=3.2.0,<3.5.0
```

Reply:
Wait, this is the wrong place. The `spacy` requirement should be in `project.yml`.

Reply:
I think this is valid. This is the fallback spaCy version. If the first project doesn't specify `spacy_version` in its `project.yml`, then we would have a python interpreter without `spacy` with the current version of `run-tests.sh`. If it does, it's overwritten anyway. There are other ways to ensure we have a valid `spacy` installation for the first project, but I think this is alright?

Reply:
Ah, I thought this was in a subdirectory, sorry. Then the CI step should install from `requirements.txt` rather than repeating this.